"./bootstrap.sh --local" fails

##### Brief Description

Regression: calling "./bootstrap.sh --local" fails since commit ead2152d6f0d15f7e662975fb3ed8525109794c3

OS: GNU/Linux (Debian 9)
qTox version: git
Commit hash:  ae350a648366f196008f21504f5579e8730cb187
toxcore:  0.2.2
Reproducible: Always

##### Steps to reproduce

1. git clone the repository
2. ./bootstrap.sh --local

##### Observed Behavior

Fails at installation with:

[100%] Built target auto_set_name_test
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/lib/libtoxcore.so.2.2.0
CMake Error at cmake_install.cmake:48 (file):
  file INSTALL cannot copy file
  "/home/.../qTox/libs/libtoxcore-v0.2.2/libtoxcore.so.2.2.0"
  to "/usr/local/lib/libtoxcore.so.2.2.0".


Makefile:104: recipe for target 'install' failed
make: *** [install] Error 1

##### Expected Behavior

With the --local option it should install to a directory inside the source tree, but instead it attemps to install to a system directory

##### Additional Info

The bug seems to be introduced with commit:
https://github.com/qTox/qTox/commit/ead2152d6f0d15f7e662975fb3ed8525109794c3
Before that commit, or the last release (v1.15.0) still worked.
toxcore should be configured to install to "${BASE_DIR}" when --local is set.

--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -93,16 +93,16 @@ install_toxcore() {
             "${BASE_DIR}/${TOXCORE_DIR}"
 
         pushd ${BASE_DIR}/${TOXCORE_DIR}
-        cmake .
 
-        # compile
-        make -j $(nproc)
-
-        # install
+        # compile and install
         if [[ $SYSTEM_WIDE = "false" ]]
         then
+            cmake . -DCMAKE_INSTALL_PREFIX=${BASE_DIR}
+            make -j $(nproc)
             make install
         else
+            cmake .
+            make -j $(nproc)
             sudo make install
             sudo ldconfig
         fi

